website/project/model.py

```diff
@@ -3563,3 +3572,159 @@ def _on_reject(self, user, token):
             },
             auth=Auth(user),
         )
+
+class DraftRegistration(AddonModelMixin, StoredObject):
+
+    is_draft_registration = True
+
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+
+    datetime_initiated = fields.DateTimeField(auto_now_add=True)
+    datetime_updated = fields.DateTimeField(auto_now=True)
+
+    branched_from = fields.ForeignField('node')
```

Comment:
Per flowdock discussion: the meaning of this nomenclature isn't clear to an unfamiliar reader. Perhaps add a comment defining what this field is?